### PR TITLE
Removes mass-spectrometers

### PR DESCRIPTION
**Pull request by ACCount12**
They weren't very useful before, but adding reagent scan mode to a regular health scanner put them on the entire new level of uselessness.

:cl: ACCount
del: Mass-spectrometers are removed. Would anyone notice if not for this changelog entry?
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
Got no complaints now that rnd is gone

**Comment by Bawhoppen** [comment]
This has its place as being a worse version of default health scanners.

**Review by Cyberboss — DISMISSED** [review]

**Comment by imsxz** [comment]
All I've ever seen them used for was RND. Nobody will miss them.

**Comment by WJohn** [comment]
And now that RND has been changed they don't even have a purpose there!

**Comment by jughu** [comment]
No let the detective keep it to scan chemicals lmfao

**Comment by ACCount12** [comment]
Detective may use health scanner, like every normal person.

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -386,67 +384,6 @@ MASS SPECTROMETER
...
-		var/list/blood_traces = list()
-		for(var/datum/reagent/R in reagents.reagent_list)
-			if(R.id != "blood")
-				reagents.clear_reagents()
-				to_chat(user, "<span class='warning'>The sample was contaminated! Please insert another sample.</span>")
-				return
-			else
-				blood_traces = params2list(R.data["trace_chem"])
-				break
-		var/dat = "<i><b>Trace Chemicals Found:</b>"
```
Comment (as334) [inline code/game/objects/items/devices/scanners.dm]: AFAIK the health sensor DOES NOT recreate this functionality(scanning a blood sample for what chems **used** to be in it), maybe just add this behavior to the detective scanner? It's still useful for a detective to try and determine if someone was poisoned, even after all that poison might be out of the body.

Comment (ACCount12) [reply code/game/objects/items/devices/scanners.dm]: Trace chems store the set of chems that were in the body when the blood sample was taken. If the toxin is out of the body already, it wouldn't help.

**Comment by RandomMarine** [comment]
<p align="center">
  <b>R.I.P.</b><br>mass spectrometer<br>2011-2017<br>slain by power creep<br>forgotten and unused
</p>

**Comment by jughu** [comment]
But if the detective have to use a health scanner hes eithee forced to drink the poison or feed it to the prisoner which us bullshit

**Comment by ACCount12** [comment]
Surprise: mass-spectrometer can't scan pills and beakers.

**Comment by QualityVan** [comment]
Having inferior tools is important.

**Comment by jughu** [comment]
IMPROVE DONT REMOVE

**Comment by DoctoreMad** [comment]
>Improve don't remove

You know this catchphrase makes me wish we had more idea guys in a way. At least they are being more productive than Jughu.

**Comment by Noahnoah55** [comment]
@DoctoreMad As a professional ideas guy, I recommend letting the mass spectrometer scan reagents in any container. Would be helpful for identifying poisoned food and the like.

**Comment by XDTM** [comment]
Currently only the sci pda cartridge can scan chems inside stuff, that would be a nice niche to give spectrometers (only chem names for base version, name and amount for advanced)

**Comment by Cyberboss** [comment]
The science goggles do that too

**Comment by XDTM** [comment]
Not for food and other closed containers, i think

**Comment by ACCount12** [comment]
I plan to add chem scanning to regular analyzers, as per WJohn's idea. We don't need a special device for it.

**Comment by RandomMarine** [comment]
Analyzers are extremely common. Poisoned food would never work then because everyone would have easy access to a reagent scanner.

**Comment by ACCount12** [comment]
The fuck is with those tags?

**Pull request merged**